lightning/src/events/mod.rs

```diff
@@ -466,9 +466,9 @@ impl_writeable_tlv_based_enum_upgradable!(ClosureReason,
 	},
 );
 
-/// Intended destination of a failed HTLC as indicated in [`Event::HTLCHandlingFailed`].
+/// The type of HTLC that is being handled in [`Event::HTLCHandlingFailed`].
```

Comment:
"the type of htlc that is being handled", but isn't it more the outcome of the htlc forward, not saying much about the htlc itself? In particular `ForwardFailed` doesn't have much to do with the htlc?

Reply:
>"the type of htlc that is being handled", but isn't it more the outcome of the htlc forward, not saying much about the htlc itself? 

Would "the handling action taken" be better?  Aiming to represent what we were supposed to do to the HTLC we were handed (forward it/ receive it / we don't know because it's invalid).

Open to better naming/wording if you have it, but I do think that `outcome` is more associated with settle/fail so not a good fit here.

> ForwardFailed doesn't have much to do with the htlc?

Do you mean `InvalidForward`?

Reply:
I think the type name `HTLCHandlingType` is fine. You could change the comment to something like 'The type of HTLC handling performed', to make it clear that it isn't about the type of the HTLC, but the type of the handling. I'd avoid also introducing the word action, or change all the 'handling' to 'action'.

Yes, `InvalidForward` indeed.